src/plugins/kv-analytics-plugin.js

```diff
@@ -331,6 +337,25 @@ export default {
 					event: 'refresh'
 				});
 			},
+			trackOPTransaction: transactionData => {
+				window.optimizely.push({
```

Comment:
I think these events should only fire when the deposit total/loan total is greater than 0, and the only tag that's needed would be the deposit amount/loan share purchase amount.